sysctl.d/meson.build

```diff
@@ -6,6 +6,19 @@ install_data(
 
 in_files = []
 
+cpu_arch = target_machine.cpu_family()
+
+if (cpu_arch == 'aarch64' or
+    cpu_arch == 'mips64' or
+    cpu_arch == 'parisc' or
+    cpu_arch == 'ppc64' or
+    cpu_arch == 'riscv64' or
+    cpu_arch == 's390x' or
+    cpu_arch == 'sparc64' or
+    cpu_arch == 'x86_64')
```

Comment:
The kernel checks the size of long:
```
#define PID_MAX_LIMIT (CONFIG_BASE_SMALL ? PAGE_SIZE * 8 : \
        (sizeof(long) > 4 ? 4 * 1024 * 1024 : PID_MAX_DEFAULT))
```
Maybe we should do the same, to avoid having the list of arches:
```
if cc.sizeof('long') > 4:
        install_data('50-pid-max.conf', install_dir : sysctldir)
endif
```
?